service/src/main/java/uk/gov/hmcts/reform/fpl/model/HearingBooking.java

```diff
@@ -80,4 +80,8 @@ public String toLabel(String dateFormat) {
         }
         return list;
     }
+
+    public boolean isHearingOfType(HearingType hearingType) {
```

Comment:
minor: is it really worth having method for this? maybe better name would be isOfType(HearingType)

Reply:
Reads nicely :)